src/middleware.ts

```diff
@@ -112,6 +113,9 @@ export function initializeMiddleware(app) {
   app.use(localsMiddleware)
   app.use(sameOriginMiddleware)
 
+  // Mount MP Graphql proxy and cache
+  app.use("/api/metaphysics", graphqlCacheProxyMiddleware)
```

Comment:
I would make this dependent on the config. The proxy is a big change to how Force operates, and there's a significant chance we need to turn it off quickly, which the config can help.

Reply:
Will do 👍 One sec...